internal/services/compute/linux_virtual_machine_resource.go

```diff
@@ -417,8 +412,22 @@ func resourceLinuxVirtualMachine() *pluginsdk.Resource {
 				Type:     pluginsdk.TypeString,
 				Computed: true,
 			},
+			"vm_agent_platform_updates_enabled": {
+				Type:     pluginsdk.TypeBool,
+				Computed: true,
+			},
 		},
 	}
+
+	if !features.FivePointOh() {
+		resource.Schema["vm_agent_platform_updates_enabled"] = &pluginsdk.Schema{
+			Type:     pluginsdk.TypeBool,
+			Optional: true,
+			Computed: true,
+		}
+	}
```

Comment:
Can we add a deprecation message here to help users that bump into this during a plan/apply understand the problem?
```suggestion
	if !features.FivePointOh() {
		resource.Schema["vm_agent_platform_updates_enabled"] = &pluginsdk.Schema{
			Type:       pluginsdk.TypeBool,
			Optional:   true,
			Computed:   true,
			Deprecated: "this property has been deprecated due to a breaking change in the API/Service making it read-only",
		}
	}
```

Reply:
Fixed.